src/NpapiCore/NpapiBrowserHost.cpp

```diff
@@ -911,6 +911,7 @@ NPObject * NpapiBrowserHost::getPromiseObject() {
 }
 
 NPObject* NpapiBrowserHost::makeError(const std::exception& e) {
+    std::cerr << "makeError: " <<e.what() << std::endl;
```

Comment:
We should not be writing directly to stderr anywhere inside  FireBreath core code; instead, please use FBLOG_ERROR to use the logging framework.